### PR TITLE
Added button generator

### PR DESCRIPTION
**Pull request by itssgokul**
Added button generator

### CODE REVIEW

File: services/resources.json [hunk 1]
```diff
@@ -278,5 +278,13 @@
     "description": "Code Beautifier Beautify CSS, HTML and JS code.",
     "link": "https://freecodetools.org/beautifier/",
     "license": "Free"
+  },
+  {
+    "id": 36,
+    "category": "Tools",
+    "title": "Buttons Generator",
+    "description": "Generat Buttons to Your Web Site",
```
Comment (anburocky3) [inline services/resources.json:286]: Spelling mistake in the description, fix that and commit once again.

**Review by anburocky3 — CHANGES_REQUESTED** [review]

**Review by anburocky3 — APPROVED** [review]

**Comment by anburocky3** [comment]
@itssgokul It is not merged and you have deleted the pull request. 

You need to send as another pull request to merge those data to the app.

**Comment by itssgokul** [comment]
> @itssgokul It is not merged and you have deleted the pull request.
> 
> You need to send as another pull request to merge those data to the app.

sorry bro....i don't know how to do this.....

**Comment by anburocky3** [comment]
> > @itssgokul It is not merged and you have deleted the pull request.
> > You need to send as another pull request to merge those data to the app.
> 
> sorry bro....i don't know how to do this.....

Fork again and redo the work and send me a fresh Pull request.

**Pull request closed without merging**